editor/src/app/login/login.component.ts

```diff
@@ -27,9 +27,11 @@ import { PopupService } from '../popup/popup.service';
     <ng-template #logInForm>
       <div *ngIf="appState.isBertaHosting" class="form-group social-login">
         <a href="{{ appState.loginUrl }}?remote_redirect={{ appState.authenticateUrl }}&amp;provider=facebook" class="button facebook">
-          Log in with Facebook</a>
+        <bt-icon-facebook></bt-icon-facebook>
```

Comment:
I'm not sure if need to extract it as component if it's simply a piece of svg code and not even used somewhere else.

Reply:
@vteivans said it's good to use a component here

Reply:
No problem, let's be consistent and use component for google as well.